packages/design-system/src/components/Button/Button.js

```diff
@@ -69,10 +70,17 @@ const Button = React.forwardRef((props, ref) => {
       aria-expanded={ariaExpanded}
     >
       {loading && (
-        <Loader size="small" className={styles[`${baseClass}__loader`]} />
+        <div className={styles[`${baseClass}__loader`]}>
+          <Loader size="small" />
```

Comment:
https://livechat.github.io/design-system/#/Components/Loader?id=loader-1 - on the bottom of the page you can find a loader with label:
```suggestion
          <Loader size="small" label={loaderLabel} labelClassName={styles[`${baseClass}__loader-label`]} />
```
Could you use it?  Or if it's hard there is LoaderLabel component which could be used instead of this custom div element